services/api/tests/src/test_admin_org.py

```diff
@@ -217,6 +219,8 @@ def test_modify_user_success(mock_pgquery, mock_check_safe_input):
         call(admin_org_data.modify_org_remove_user_sql),
         call(admin_org_data.modify_org_add_rsu_sql),
         call(admin_org_data.modify_org_remove_rsu_sql),
+        call(admin_org_data.modify_org_add_intersection_sql),
```

Comment:
Just noticing with these additions, but we're calling admin_org.modify_org and the test method is called "test_modify_user_success". Should we rename to reflect what we're really doing here?

Reply:
re-named